lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +184,40 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        if re.fullmatch(r'/novosti/.+', article_bs['href']):
+            return 'https://www.vgoroden.ru' + article_bs['href']
+
+
 
     def find_articles(self) -> None:
         """
         Finds articles
         """
-        pass
+        #считывает только 36 ссылок, нужно исправить
```

Comment:
1. put as minimal number of comments as possible
2. forget russian when making comments in code (!)